src/js/modules/infragistics.ui.upload.js

```diff
@@ -1545,7 +1545,22 @@
 			*/
 			fileSizeDecimalDisplay: 2,
 			/* type="int" Maximum size(in bytes) allowed for the file to be uploaded. If it is set to null or -1 there is no limitation otherwise if the size(of the selected file) exceeds this value it is not allowed to be uploaded. This option is used for validation only on client side and only if the browser supports HTML5 file API and share information about the file size */
-			maxFileSize: null
+			maxFileSize: null,
+			/* type="bool" Get or set whether to use only one request for sending data, when you are sending more than one file.
+			```
+				//Initialize
+				$(".selector").igUpload({
+					sendDataInOneRequest : true
+				});
+
+				//Get
+				var sendDataInOneRequest = $(".selector").igUpload("option", "sendDataInOneRequest");
+
+				//Set
+				$(".selector").igUpload("option", "sendDataInOneRequest", true);
+			```
+			*/
+			sendDataInOneRequest: false
```

Comment:
Maybe we should name it `singleRequest` or `useSingleRequest`? @SlavUI What do you think?

Reply:
I'm for "singleRequest".

Reply:
I vote for useSingleRequest as it implies what value is expected.

Reply:
Let's call it useSingleRequest :)